text_extensions_for_pandas/io/test_watson.py

```diff
@@ -0,0 +1,879 @@
+#
+#  Copyright (c) 2020 IBM Corp.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+import textwrap
+import unittest
+
+from text_extensions_for_pandas.io.watson import *
+
+'''
+text = textwrap.dedent(
+  """\
+  Who is General Failure and why is he reading my hard disk?
+  If Barbie is so popular, why do you have to buy her friends?""")
+'''
+response1 = {
```

Comment:
You might want to put this data into a JSON file somewhere in the repository instead of an inline Python literal.